cirq/google/xmon_device_test.py

```diff
@@ -55,6 +55,15 @@ def test_init():
         _ = d.duration_of(cirq.Gate().on(q00))
 
 
+@cirq.testing.only_test_in_python3
+def test_repr():
+    d = square_device(2, 2, holes=[])
+
+    assert repr(d) == ("XmonDevice(measurement_duration=Duration(picos=1000), "
```

Comment:
Almost there! Just need to also print out the qubits. Add `qubits={}` with value `sorted(self.qubits)` to the output. :+1: